elements/pl-units-input/pl-units-input.mustache

```diff
@@ -0,0 +1,188 @@
+            {{#suffix}}
+            <span class="input-group-text {{^show_info}}rounded-right{{/show_info}}" id="pl-units-input-{{uuid}}-suffix">
+                {{suffix}}
+            </span>
+            {{/suffix}}
+
+            {{#show_info}}
+            <a role="button" class="btn btn-light border d-flex align-items-center"
+                data-toggle="popover" data-html="true" title="Units" data-content="{{info}}" data-placement="auto" data-trigger="focus" tabindex="0">
+                <i class="fa fa-question-circle" aria-hidden="true"></i>
+            {{/show_info}}
+            {{^show_info}}
+            <span class="d-flex align-items-center">
+            {{/show_info}}
+
+            {{#correct}}
+            &nbsp;
+            <span class="badge badge-success">
+                <i class="fa fa-check" aria-hidden="true"></i> 100%
+            </span>
+            {{/correct}}
+            {{#partial}}
+            &nbsp;
+            <span class="badge badge-warning">
+                <i class="far fa-circle" aria-hidden="true"></i> {{partial}}%
+            </span>
+            {{/partial}}
+            {{#incorrect}}
+            &nbsp;
+            <span class="badge badge-danger">
+                <i class="fa fa-times" aria-hidden="true"></i> 0%
+            </span>
+            {{/incorrect}}
+
+            {{#show_info}}</a>{{/show_info}}
+            {{^show_info}}</span>{{/show_info}}
+        </span>
+    </span>
+{{#inline}}</span>{{/inline}}
+{{/question}}
+
+
+{{#submission}}
+{{#inline}}<span class="d-inline-block">{{/inline}}
+
+{{#error}}
+<script>
+    $(function(){
+        $('#pl-units-input-submission-{{uuid}} [data-toggle="popover"]').popover({
+            sanitize: false,
+            container: 'body',
+            template: '<div class="popover pl-units-input-popover" role="tooltip"><div class="arrow"></div><h3 class="popover-header"></h3><div class="popover-body"></div></div>',
+        });
+    });
+</script>
+
+<span id="pl-units-input-submission-{{uuid}}">
+    {{#parse_error}}
+    {{#label}}<span>{{{label}}}</span>{{/label}}
+    {{#raw_submitted_answer}}<code class="user-output-invalid">{{raw_submitted_answer}}</code>{{/raw_submitted_answer}}
+    <span class="badge text-danger badge-invalid">
+        <i class="fa fa-exclamation-triangle" aria-hidden="true"></i> Invalid
+    </span>
+    <a href="javascript:void(0);" role="button" class="btn btn-sm btn-secondary small border"
+        data-placement="auto" data-trigger="focus" data-toggle="popover" data-html="true"
+        style="margin-left: 5px" title="Format Error" tabindex="0" data-content="{{parse_error}}">
+        Why
+        <i class="fa fa-question-circle" aria-hidden="true"></i>
+    </a>
+    {{/parse_error}}
+
+    {{#missing_input}}
+    {{#label}}<span>{{{label}}}</span>{{/label}}
+    <span class="badge text-dark badge-missing-input">
+        <i class="fa fa-question-circle" aria-hiden="true"></i> Missing Input
```

Comment:
aria-hidden is misspelled here

Reply:
Fixed in [4d5b245](https://github.com/PrairieLearn/PrairieLearn/pull/6790/commits/4d5b245479e0260867e69994f65d9d5b8dd71cdb).